src/ngraph/ngraph_sgcompiler.cc

```diff
@@ -86,8 +76,11 @@ void CompileForwardBackward(std::shared_ptr<Graph> sub_graph,
 
   // Log the graphs so Graph_* corresponds to Function_* in codgen
   if (ngraph_log_graph) {
-    dump_graph(f_copy);
-    dump_graph(bf_copy);
+    gdumper.dump(f_copy);
+    gdumper.increment_serial_num();
```

Comment:
Would it make sense to put the increment in the dump function do avoid so many calls here?

Reply:
I thought about it, but the semantics was getting messy.  

If/when we create additional graph dumps that have no corresponding ngraph JIT compilation, we'll have even more "dump" sites where we _don't_ want to increment.

I'm not deeply passionate about this.  If you want them rolled together, lemme know.

Reply:
Naw